### ISSUE TITLE
Uhh can someone tell me what the modip does?

### ISSUE THREAD

**Issue by hiimjustin000** [Cvolton/GMDprivateServer#609]
I want to know.

**Comment by Wyliemaster** [comment]
not entirely sure with this as i haven't used it personally but, ive been told it works with modipperms to basically let mods get free copies on levels. i could be wrong as i havent really looked into it though

**Comment by Cvolton** [comment]
yes, that's the only thing it does nowadays, back when account checks weren't properly implemented, all mod actions relied on this
